content/Day-6/_index.md

```diff
@@ -0,0 +1,94 @@
+
+### 5. Identify Key Points or Objectives
+
+The purpose of this document is to reduce the Mean Time To Respond (MTTR) in password reset calls with a document that captures the required steps for Technical Analyst to complete this task. 
+
+## Learner Profile (Technical, non-technical)
+
+A learner's profile (aka target audience) is another approach for technical writing in that we are often writing for both technical and non-technical users. This means we must take a different approach when explaining complex technical designs and design presentations if our learner's profile is for a non-technical reader. We should also stay away from using industry jargon, which are the standard words and terminology used by professionals in the particular field. For example, a sales team would use words that make sense to other sales people but may not mean the same thing if that sales person was talking to a software engineer and vice versa. This means our technical writing should include charts and other visual aids to help express our message. We may be writing for developers, engineers or executives leadership teams (Vice President (VP), Chief Executive Officer (CEO) etc.). We should ensure our audience can keep up with the key points of the presentation and not get hung up on words and terms we have used to explain a concept.
+
+1. Will anyone in your audience have preexisting knowledge of the content in your presentation? 
+2. Will you have any Subject Matter Experts (SME’s) in the room? 
+
+If either of the previous questions are true, we have to addressed their potential concerns or invited the SME to help with the presentation. We need to know how to best tell our story rather than provide bullet points and text on the screen or in an email. Another consideration is with font-size. We want our font to be readable and consider that some team members may be color blind or wear glasses. That means if we highlight text in a particular color like red. We may want to ensure we consider other colors or explain what the red color represents in our communication to the team. There are many types of readers to consider and we do not have to try to remember them all but we should be familiar with accessibility standards. You can read more about [web accessibility](https://www.w3.org/WAI/standards-guidelines/wcag/) for web based documentation but also should be aware of the other types of accessibility in reference to documentation that we create for mass consumption. Some accessibility areas include but is not limited to:  
+
+* Font-Size 
+* Colors 
+* Language 
+* Proper Headings 
+* Text Readability 
+* Image alternate text
+
+## Presentation and Visual Impact 
+
+> “ I hate it when people use PowerPoints instead of thinking. People confront a problem by creating a presentation. I want them to engage,  hash things out at the table, rather than show a bunch of slides. People who know what they’re talking about don’t need PowerPoint.” – Steve Jobs
```

Comment:
I am not sure about this quote. In addition to many people now having a strong dislike for Steve Jobs, it does not set the stage for why we are teaching how to effectively present information.

Reply:
This is meant to be an example reference for some companies like Apply and Amazon that banded powerpoints in the event the learners see this latter as added value or want to do something different they can choose this route of a documented plan vs powerpoint decks